src/maintenance/azext_maintenance/generated/commands.py

```diff
@@ -9,84 +9,138 @@
+        'azext_maintenance.vendored_sdks.maintenance.operations._apply_updates_operations#ApplyUpdatesOperations.{}'
+    ),
+    client_factory=cf_apply_update,
+)
+
+
+maintenance_configuration_assignment = CliCommandType(
+    operations_tmpl='azext_maintenance.vendored_sdks.maintenance.operations._configuration_assignments_operations#ConfigurationAssignmentsOperations.{}',
+    client_factory=cf_configuration_assignment,
+)
+
+
+maintenance_maintenance_configuration = CliCommandType(
+    operations_tmpl='azext_maintenance.vendored_sdks.maintenance.operations._maintenance_configurations_operations#MaintenanceConfigurationsOperations.{}',
+    client_factory=cf_maintenance_configuration,
+)
+
+
+maintenance_configuration_assignment_within_subscription = CliCommandType(
+    operations_tmpl='azext_maintenance.vendored_sdks.maintenance.operations._configuration_assignments_within_subscription_operations#ConfigurationAssignmentsWithinSubscriptionOperations.{}',
+    client_factory=cf_configuration_assignment_within_subscription,
+)
+
+
+maintenance_maintenance_configuration_for_resource_group = CliCommandType(
+    operations_tmpl='azext_maintenance.vendored_sdks.maintenance.operations._maintenance_configurations_for_resource_group_operations#MaintenanceConfigurationsForResourceGroupOperations.{}',
+    client_factory=cf_maintenance_configuration_for_resource_group,
+)
+
+
+maintenance_public_maintenance_configuration = CliCommandType(
+    operations_tmpl='azext_maintenance.vendored_sdks.maintenance.operations._public_maintenance_configurations_operations#PublicMaintenanceConfigurationsOperations.{}',
+    client_factory=cf_public_maintenance_configuration,
+)
+
+
+maintenance_update = CliCommandType(
+    operations_tmpl='azext_maintenance.vendored_sdks.maintenance.operations._updates_operations#UpdatesOperations.{}',
+    client_factory=cf_update,
+)
 
 
 def load_command_table(self, _):
 
-    from azext_maintenance.generated._client_factory import cf_public_maintenance_configuration
-    maintenance_public_maintenance_configuration = CliCommandType(
-        operations_tmpl='azext_maintenance.vendored_sdks.maintenance.operations._public_maintenance_configurations_oper'
-        'ations#PublicMaintenanceConfigurationsOperations.{}',
-        client_factory=cf_public_maintenance_configuration)
-    with self.command_group('maintenance public-configuration', maintenance_public_maintenance_configuration,
-                            client_factory=cf_public_maintenance_configuration) as g:
-        g.custom_command('list', 'maintenance_public_configuration_list')
-        g.custom_show_command('show', 'maintenance_public_configuration_show')
+    with self.command_group(
+        'maintenance apply-update-for-resource-group',
+        maintenance_apply_update_for_resource_group,
+        client_factory=cf_apply_update_for_resource_group,
+    ) as g:
+        g.custom_command('list', 'maintenance_apply_update_for_resource_group_list')
```

Comment:
Swagger: https://github.com/Azure/azure-rest-api-specs-pr/blob/main/specification/maintenance/resource-manager/Microsoft.Maintenance/preview/2021-09-01-preview/Maintenance.json

Reply:
hide new operation from cli since these are not implemented in backend.
```
(venv) root@320dfdeb8b53:/home/azext# az extension show -n maintenance -o table
ExtensionType    Name         Path                         Version
---------------  -----------  ---------------------------  ---------
dev              maintenance  /home/azext/src/maintenance  1.3.0
(venv) root@320dfdeb8b53:/home/azext# az maintenance --help

Group
    az maintenance : Manage Maintenance.
        This command group is experimental and under development. Reference and support levels:
        https://aka.ms/CLI_refstatus
Subgroups:
    applyupdate                     : Manage apply update with maintenance.
    assignment                      : Manage configuration assignment with maintenance.
    configuration                   : Manage maintenance configuration with maintenance.
    public-configuration [Preview]  : Manage public maintenance configuration with
                                      maintenance.
    update                          : Manage update with maintenance.

For more specific examples, use: az find "az maintenance"

Please let us know how we are doing: https://aka.ms/azureclihats
``